src/opm/parser/eclipse/EclipseState/Aquifetp.cpp

```diff
@@ -0,0 +1,57 @@
+/*
+  Copyright (C) 2017 TNO
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <opm/parser/eclipse/EclipseState/Aquifetp.hpp>
+
+
+namespace Opm{
+	
+  Aquifetp::Aquifetp(const EclipseState& eclState, const Deck& deck)
```

Comment:
Does not seem like the `eclState` parameter is used at all - I guess the constructor could have signature:
```
Aquifetp::Aquifetp(const Deck& deck);
```